toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -117,6 +124,23 @@ done
 # Turn back on echoing input so the TTY session is usable for the user.
 stty echo
 
+
+# TODO: CHECK IF \EFI\BOOT\bootx64.efi is already first in boot order, if so, replace it?
+if [[ -f "./attendedconfig.json" ]]; then  #if iso is an attended install
+    if [[ $(jq ".SystemConfigs[0].BootType" ./attendedconfig.json -r) == "efi" ]]; then # if boot type is efi
+        TARGET_DISK=$(jq ".Disks[0].TargetDisk.Value" ./attendedconfig.json | tr -d '"')
+        BOOT_PART_IDX=$(jq '.Disks[0].Partitions as $parts | $parts | index($parts[] | select(.Flags | index("boot")))' ./attendedconfig.json) #finds the index of the partition with "boot" in it's Flags array
+        BOOT_PART_NO=$(($BOOT_PART_IDX+1)) 
+    fi
```

Comment:
This code seems identical to the one few lines above with the only difference being the JSON file path. Let's remove the code duplication by enclosing this in a function.